src/policy/policy.h

```diff
@@ -68,6 +68,7 @@ bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
 
 namespace Policy {
     std::int32_t blockSizeAcceptLimit();
+    std::int64_t blockSigOpAcceptLimit(int32_t nBlockSize);
```

Comment:
This limits the block size to 4GB. Not sure this is a major concern at this point.